CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # App Center SDK for React Native Change Log
 
+## Version 1.12.0
+
+### AppCenter
+
+* **[Fix]** Fix issue where the SDK source could not be built in a directory that contains escaped characters (applies to all modules).
+* **[Fix]** Fix TLS 1.2 configuration for some specific devices running API level <21. The bug did not affect all devices running older API levels, only some models/brands, and prevented any data from being sent.
+
+### AppCenterAnalytics
+
+* **[Feature]** Preparation work for a future change in transmission protocol and endpoint for Analytics data. There is no impact on your current workflow when using App Center.
```

Comment:
remove that feature it was elusive change for macos userId support, our react SDK does not have macos support.

Reply:
done